src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlBulkCopy.cs

```diff
@@ -1202,6 +1206,19 @@ private bool ReadFromRowSource()
 
         private SourceColumnMetadata GetColumnMetadata(int ordinal)
         {
+            bool IsMetadataDataStream(_SqlMetaData metadata)
```

Comment:
I opted to rewrite this using a local function to make switching in the netfx-only check cleaner